src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -523,4 +630,27 @@ public static boolean canPlayerUseItem(@Nonnull Player p, @Nullable ItemStack it
     public static @Nullable Item spawnItem(Location loc, ItemStack item, ItemSpawnReason reason) {
         return spawnItem(loc, item, reason, false);
     }
+
+    /**
+     * Helper method to check if an Inventory is empty (has no items in "storage").
+     * If the MC version is 1.16 or above
+     * this will call {@link Inventory#isEmpty()} (Which calls MC code resulting in a faster method).
+     *
+     * @param inventory
+     *            The {@link Inventory} to check.
+     *
+     * @return True if the inventory is empty and false otherwise
+     */
+    public static boolean isInventoryEmpty(@Nonnull Inventory inventory) {
+        if (Slimefun.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_16)) {
+            return inventory.isEmpty();
+        } else {
+            for (ItemStack is : inventory.getStorageContents()) {
+                if (is != null && !is.getType().isAir()) {
```

Comment:
Can we give this variable a better name?

Reply:
Out of scope - I know this appears in my diff but has not been worked on as part of this PR, just moved around. I don't want more in here than is necessary.